chsdi/lib/validation/mapservice.py

```diff
@@ -235,3 +245,15 @@ def offset(self, value):
                 raise HTTPBadRequest('Please provide an integer as an offset parameter')
             else:
                 self._offset = int(value)
+
+    @limit.setter
+    def limit(self, value):
+        if value is not None:
+            if value.isdigit() and int(value) >= 0:
+                self._limit = int(value)
+            else:
+                raise HTTPBadRequest('Please provide a positive integer for the parameter limit')
+
+    @order.setter
+    def order(self, value):
+        self._order = value
```

Comment:
Additionally it should work only if a  spatial filter is defined. (e.g. the parameter geometry is defined)
I know it's the case in the code but could make a check here.